pkg/kubelet/lifecycle/predicate.go

```diff
@@ -28,6 +28,9 @@ import (
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
+// ErrNodeInfoCacheNotReady is used to describe the error that nodeInfo is not ready
+const ErrNodeInfoCacheNotReady string = "nodeInfo is not ready"
```

Comment:
Would it be possible for this to share the value with `kubelet/errors.go` so that we don't see drift between the two names?

Reply:
Import cycle error will happen.

Reply:
Import cycle err will happen.